css/FAQ.css

```diff
@@ -0,0 +1,13 @@
+h1{
+	text-align:center;
+}
+
+.headingalignment{
+	margin-left: 5%;
+}
+.sectionalignment{
+	margin-left: 10%;
```

Comment:
Let's use more descriptive names? "about-us", "questions-group".

What if in future we have more headings, we will not be able to use any names there.

Reply:
topic and question-group. Will it work ?